torch/ao/quantization/qconfig.py

```diff
@@ -325,6 +328,12 @@ def get_default_qat_qconfig(backend='fbgemm', version=1):
                                                                 quant_min=0,
                                                                 quant_max=255),
                               weight=default_per_channel_weight_fake_quant)
+        if backend == 'x86':
+            qconfig = QConfig(activation=FakeQuantize.with_args(observer=MovingAverageMinMaxObserver,
+                                                                quant_min=0,
+                                                                quant_max=255,
+                                                                reduce_range=True),
```

Comment:
please don't use `reudce_range`, and change `quant_min`/`quant_max` directly

Reply:
Thanks. It's fixed now.